extensions/wikia/AdEngine/js/lookup/prebid/adapters/appnexusPlacements.js

```diff
@@ -1,19 +1,71 @@
 /*global define*/
 define('ext.wikia.adEngine.lookup.prebid.adapters.appnexusPlacements', [
 	'ext.wikia.adEngine.adContext',
-	'wikia.instantGlobals',
 	'wikia.log'
-], function (adContext, instantGlobals, log) {
+], function (adContext, log) {
 	'use strict';
 
 	var logGroup = 'ext.wikia.adEngine.lookup.prebid.adapters.appnexusPlacements',
-		placementsMap = instantGlobals.wgAdDriverAppNexusBidderPlacementsConfig;
+		placementsMap = {
+			'mercury': {
+				'ent': '9412992',
+				'gaming': '9412993',
+				'life': '9412994',
+				'other': '9412994'
+			},
+			'oasis': {
+				'ent': {
+					'atf': '9412983',
+					'btf': '9412986',
+					'hivi': '9412989'
+				},
+				'gaming': {
+					'atf': '9412984',
+					'btf': '9412987',
+					'hivi': '9412990'
+				},
+				'life': {
+					'atf': '9412985',
+					'btf': '9412988',
+					'hivi': '9412991'
+				},
+				'other': {
+					'atf': '9412985',
+					'btf': '9412988',
+					'hivi': '9412991'
+				},
+				'recovery': {
+					'atf': '11823778',
+					'btf': '11823724',
+					'hivi': '11823799'
+				}
+			}
+		},
+		palPlacementsMap = {
```

Comment:
Can we move it to `oasis` object? So we'll have `ent`, `gaming`, `life`, `other`, `recovery`, `pal`